Cangjie5.txt

```diff
@@ -13734,6 +13734,7 @@
 渓	ebqo
 淍	ebqr
 泂	ebr
+𪸅	ebrf
```

Comment:
We already have
```
𪸅	erhvf
```
I don't think this mapping is wrong but it will be great if we can align with
```
𣠡	dbrf
𢑈	nbrf
𡈸	wbrf
邎	ybrf
```

Reply:
`𪸅	erhvf` is wrong.

Reply:
Okay I can remove this one.